Sources/View/Abstract/BaseView.swift

```diff
@@ -11,9 +11,9 @@ import UIKit
 /**
  BaseView acts as a common base for all custom views
  */
-@objc public class BaseView: UIView {
+@objc open class BaseView: UIView {
```

Comment:
Do you have plans to use our open source version of this library  / convention now it's live and has CocoaPods support? Maybe one for later